meta/happy_hour.md

```diff
@@ -1,25 +1,32 @@
 # Happy Hours at Artsy
 
-Happy Hours are a long-standing event at Artsy where employees get together with our partners, artists and fellow engineers.
+Happy Hours are a long-standing event at Artsy where employees get together and invite our partners, friends, artists and fellow engineers.
 
-It's held inside our offices, on a weekly basis. We strive to make them full of people across the tech/arts spectrum. 
+It's held inside our offices, on a weekly basis. We strive to make them full of people across the tech & arts spectrum. Other than friends we tend to invite great candidates that we might not be currently interested in, but maybe will be in the future.
 
-There are small snacks, drinks ranging from beer to wine. 
+<a href="http://www.nytimes.com/2013/06/20/fashion/the-calorie-packed-perk.html" target="_blank">![happy hour, via the New York Times](images/happy_hour.jpg)</a>
 
-## Location
-
-![Map of NYC][map]
+There are small snacks, drinks (including beer, wine, whiskey, etc.). We have a bartender.
 
-Arrive at Artsy for 6:00pm ( We're [401 Broadway, 24th Floor, 10013][401] - Canal St is the closest station)
+## Location
 
+<a href="(https://www.google.com/maps/place/401+Broadway/@40.718958,-74.0049492,17z/data=!3m1!4b1!4m5!3m4!1s0x89c2598a7196824f:0xddf53435afbdd5b9!8m2!3d40.718954!4d-74.0027552)">![Map of NYC](https://user-images.githubusercontent.com/49038/28329863-77689446-6bb9-11e7-8bd9-1805ed8bf140.png)</a>
 
 ## TLDR
 
 * Thursday in summer, Friday the rest of the time
-* 401 Broadway, closest to Canal Street station, 24th Floor,  6:00pm - 8:30pm
-* make sure that people who are coming are on the list
+* 401 Broadway, closest to Canal Street station, 24th Floor, 6-8:30pm
+* make sure that people who are coming are on the list with their first and last names
```

Comment:
Maybe this could be more like:

> * make sure that whoever invited your has your first and last name on the list, as well as others you are bringing

as it implies that someone else has to do it